README.md

```diff
@@ -71,6 +78,8 @@ The [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) spec is deliberately not dup
 - Images should always be rendered with `<img>` for screen readers and ease of copying from the right click menu
 - Illustrations built with HTML should have an explicit `aria-label` instead of announcing the raw DOM tree to people using screen readers
 - Gradient text should unset the gradient on `::selection` state
+- When using submenus, make sure projected mouse trajectories over non interactive elements don’t close the menu if moved above a certain speed threshold
```

Comment:
Does this reference [this](https://twitter.com/JohnPhamous/status/1657083267299028992) problem?

Reply:
YES. I just couldn't find an example.

Reply:
I like that more!